demo/SnowplowDemo/Analytics/Tracker.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+import SnowplowTracker
+
+public class Tracker {
+    public static let shared = Tracker()
+    private let tracker: TrackerController
+
+    init() {
+        tracker = Snowplow.createTracker(namespace: "DemoAppExtensionNamespace", endpoint: "acme.fake.com", method: .post)
+    }
+
+    public func track(event: String) {
```

Comment:
It would be great if we could propagate the `contentText` input from the extension to be tracked in the event.